conda/recipes/ucxx/recipe.yaml

```diff
@@ -0,0 +1,273 @@
+# Copyright (c) 2019-2025, NVIDIA CORPORATION.
+
+context:
+  version: ${{ env.get("RAPIDS_PACKAGE_VERSION") }}
+  cuda_version: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[:2] | join(".") }}
+  cuda_major: ${{ (env.get("RAPIDS_CUDA_VERSION") | split("."))[0] }}
+  date_string: ${{ env.get("RAPIDS_DATE_STRING") }}
+  py_version: ${{ env.get("RAPIDS_PY_VERSION") }}
+  head_rev: ${{ git.head_rev(".")[:8] }}
+
+recipe:
+  name: libucxx-split
+
+cache:
+  source:
+    path: ../../..
+
+  requirements:
+    build:
+      - cmake ${{ cmake_version }}
+      - ninja
+      - ${{ compiler("c") }}
+      - ${{ compiler("cxx") }}
+      - ${{ compiler("cuda") }}
+      - cuda-version =${{ cuda_version }}
+      - ${{ stdlib("c") }}
+    host:
+      - cuda-version =${{ cuda_version }}
+      - if: cuda_major != "11"
+        then: cuda-cudart-dev =${{ cuda_version }}
+      - rapids-build-backend >=0.3.0,<0.4.0.dev0
+      - ucx
+      - python
+      - librmm ${{ rapids_version }}
+
+  build:
+    script:
+      content:
+        - export ucxx_ROOT="$(realpath ./cpp/build)"
+        - ./build.sh -n -v libucxx libucxx_python benchmarks examples tests --cmake-args=\"-DCMAKE_INSTALL_LIBDIR=lib\"
+      secrets:
+        - AWS_ACCESS_KEY_ID
+        - AWS_SECRET_ACCESS_KEY
+        - AWS_SESSION_TOKEN
+
+      env:
+        CMAKE_C_COMPILER_LAUNCHER: ${{ env.get("CMAKE_C_COMPILER_LAUNCHER") }}
+        CMAKE_CUDA_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CUDA_COMPILER_LAUNCHER") }}
+        CMAKE_CXX_COMPILER_LAUNCHER: ${{ env.get("CMAKE_CXX_COMPILER_LAUNCHER") }}
+        CMAKE_GENERATOR: ${{ env.get("CMAKE_GENERATOR") }}
+        PARALLEL_LEVEL: ${{ env.get("PARALLEL_LEVEL") }}
+        SCCACHE_BUCKET: ${{ env.get("SCCACHE_BUCKET") }}
+        SCCACHE_IDLE_TIMEOUT: ${{ env.get("SCCACHE_IDLE_TIMEOUT") }}
+        SCCACHE_REGION: ${{ env.get("SCCACHE_REGION") }}
+        SCCACHE_S3_USE_SSL: ${{ env.get("SCCACHE_S3_USE_SSL") }}
+        SCCACHE_S3_NO_CREDENTIALS: ${{ env.get("SCCACHE_S3_NO_CREDENTIALS") }}
+        SCCACHE_S3_KEY_PREFIX: libucxx-${{ env.get("RAPIDS_CONDA_ARCH") }}
+
+outputs:
+  - package:
+      name: libucxx
+      version: ${{ version }}
+    build:
+      string: cuda${{ cuda_major }}_${{ date_string }}_${{ head_rev }}
+      script: install_libucxx.sh
+    requirements:
+      host:
+        - ucx
+        - cuda-version =${{ cuda_version }}
+        - cmake ${{ cmake_version }}
+      run:
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - ucx >=1.15.0,<1.19.0
+      run_exports:
+        - ${{ pin_subpackage("libucxx", upper_bound="x.x") }}
+      ignore_run_exports:
+        from_package:
+          - ucx
+          - python
+          - cuda-cudart-dev
+        by_name:
+          - cuda-version
+    tests:
+      - package_contents:
+          files:
```

Comment:
Can we alphabetically sort this whole list of files?

(Also we used to have lists of files to verify for cudf but we dropped it there because it was annoying to maintain. But I am okay with the status quo here.)